docs/content/interop.md

```diff
@@ -23,85 +23,30 @@ These classes:
 You need to use the ICG to generate `.schema` files and type bindings for any new native class or new Blueprint class that you want to replicate (for simplicity we’ll call these “user-defined classes”) representing new features in your game.
 
 ### How to use the ICG
-As the Interop Code Generator is an plugin for the Unreal toolbar, to use it you must first:
+As the Interop Code Generator is a plugin for the Unreal toolbar, to use it you must first:
 *  build your game project to access the SpatialOS fork of Unreal Engine as well as the Unreal GDK. (You do this as part of the [installation and setup](../setup-and-installing.md), swapping the “Starter Project” and its file location for your game’s name and file location.)
-* set up a `DefaultEditorSpatialGDK.ini` file. See  [Setting up the Interop Code Generator](#Setting-up-the-Interop-Code-Generator), below.
+* list the classes that you want to create typebindings for in the in the **SpatialOS Unreal GDK** settings in the Unreal Engine 4 Editor. For instructions on how to do this, see the guidance on [Setting up the Interop Code Generator](#Setting-up-the-Interop-Code-Generator), below.
 
 The SpatialOS build of Unreal has an **Interop Codegen** button; on the SpatialOS Unreal GDK toolbar, select **Interop Codegen** to trigger the ICG process:
 
 ![Interop Codegen button on toolbar](../assets/screen_grabs/codegen.png)
 
 ## Setting up the Interop Code Generator
 
-In order for the ICG to run correctly, you need to add any new user-defined class, along with its dependencies, to a `DefaultEditorSpatialGDK.ini` file before you run it.
+In order for the ICG to run correctly, you need to add any new user-defined class, along with its dependencies, to the **SpatialOS Unreal GDK** settings in the Unreal Engine 4 Editor tab before you run it.
 
-Using the Unreal GDK Starter Project as an example, to do this:
-1. Locate the `DefaultEditorSpatialGDK.ini` file in the Starter Project repository which you cloned during installation and setup - it’s at `Game/Config/DefaultEditorSpatialGDK.ini`.
-1. Open the file in your editor and add your user-defined class as shown in the file snippet and file description below.
+To do this:
+1. Go to ** Edit > Project Settings... > SpatialOS Unreal GDK > Toolbar **.
```

Comment:
Formatting won't work (stars in the wrong place). Also the ">" isn't bold as it isn't on screen. So it needs to be:
```
**Edit** > **Project Settings...** > **SpatialOS Unreal GDK** > **Toolbar **
```